tests/client/db.rs

```diff
@@ -101,6 +101,17 @@ fn create_and_get_users() {
     db.drop_database().unwrap();
     db.drop_all_users(None).unwrap();
 
+    let kevin_options = CreateUserOptions {
+        custom_data: None,
+        roles: vec![Role::All(AllDatabaseRole::Read)],
+        write_concern: None,
+    };
+
+    db.create_user("kevin",
+                     "ihavenosenseofhumorandthereforeihatepuns!",
```

Comment:
Haha, I had forgotten about that until I made this change, but I'm not even slightly surprised that  did this